app/configurator/components/chart-options-selector.tsx

```diff
@@ -1591,15 +1705,16 @@ const ChartFieldColorComponent = (props: ChartFieldColorComponentProps) => {
           })}
           field={field}
           path="color.componentId"
-          options={measuresOptions}
-          isOptional={optional}
+          options={options}
+          optionGroups={optionGroups}
+          optional={optional}
         />
         {enableUseAbbreviations && (
           <Box sx={{ mt: 1 }}>
             <ChartFieldAbbreviations
               field={field}
```

Comment:
This File feels a super large, can we split things or is it better to keep everything here in this file?

Reply:
Yeah, that's true 😅 I think we could split things up, I'll create a new issue for this that we can pick up maybe during Maintenance Quarterly or in case we have some "downtime", as I expect this would take a bit of time to extract all the components into respective files and wouldn't bring any new features or improvements to the end users – but for sure would make our lives easier, so we should tackle it at some point 💯

Reply:
See https://github.com/visualize-admin/visualization-tool/issues/1908 :)